docs/config-prereqs.txt

```diff
@@ -495,6 +495,154 @@ required:
+----
+:; slackpkg update
+----
+
+Note that packages may be only installed or re-installed/upgraded as separate
+explicit operations, so the procedure to bring your system into needed shape
+is a bit cumbersome (and each command may by default be interactive with a
+choice menu), e.g.:
+----
+:; for P in \
+    bash mc vim sudo \
+   ; do slackpkg info "$P" || slackpkg install $P || break ; done
+----
+
+For procedures below, this is automated via `root` profile to become a
+`slackpkg-install` command:
+----
+:; grep "slackpkg-install" ~/.profile || {
+    echo 'slackpkg-install() { for P in "$@" ; do echo "=== $P:"; slackpkg info "$P" || slackpkg install "$P" || break ; done; }' >> ~/.profile
+   }
+:; . ~/.profile
+----
+
+If something has a hiccup, it suggests to look for the right name, e.g.:
+----
+:; slackpkg search python
+----
+
+For NUT dependencies and build tools:
+----
+:; slackpkg update
+
+# Baseline toolkits:
+# Note there is no cppcheck, cppunit, valgrind...
+# Note clang compiler tools are part of llvm package
+:; slackpkg-install \
+    ccache time \
+    coreutils diffutils \
+    git python3 perl curl \
+    make autoconf automake libtool binutils \
+    pkg-config \
+    gcc llvm
+
+# For spell-checking, highly recommended if you would propose pull requests:
+:; slackpkg-install aspell{,-en}
+
+# Note there is no direct "asciidoc" nor "a2x", just the competing project
+# "rubygem-asciidoctor" that NUT currently has no recipes for, so you can
+# not compile man/html/pdf docs here, per the default repository. See below
+# for tools from alternative repositories, which seem to work well. Manpage
+# compilation would require docbook-xml resources; older versions are in
+# this package https://slackbuilds.org/repository/15.0/system/docbook-xml/
+# and recent ones are in not-installed part of main repository:
+:; slackpkg-install linuxdoc-tools
+
+# More on Python (for NUT-Monitor UI):
+:; slackpkg-install \
+    python-pip qt5 gettext-tools gettext
+
+# For CGI graph generation - massive packages (X11):
+:; slackpkg-install \
+    gd
+
+# General dependencies:
+:; slackpkg-install \
+    openssl openssl-solibs mozilla-nss \
+    libusb \
+    net-snmp \
+    neon
+
+# Shells:
+:; slackpkg-install \
+    bash dash ksh93
+----
+
+Some more packages are available on the side, including Java (useful
+e.g. to make this environment into a fully fledged Jenkins worker).
+Other common NUT dependencies absent from primary Slackware repositories
+can be found and downloaded (seek `*.txz` package files, although a few
+are named `*.tgz`) from here, and passed to `installpkg`:
+
+* http://www.slackware.com/~alien/slackbuilds/openjdk11/
+* http://www.slackware.com/~alien/slackbuilds/asciidoc/
+* http://www.slackware.com/~alien/slackbuilds/cppunit/
+* http://www.slackware.com/~alien/slackbuilds/lua/ (5.1 currently) and
+  http://www.slackware.com/~alien/slackbuilds/lua53/ -- note that LUA is not
```

Comment:
https://github.com/networkupstools/nut/pull/2143#issuecomment-1790388164

Reply:
For those forks I had in mind to pluck code from, IIRC lua-5.1 maybe 5.2 was required, but newer ones may have needed syntax adaptation :)

Still, neither is a problem for master-builds today (useful to stage the fork assimilation though).